mlir/test/CMakeLists.txt

```diff
@@ -45,6 +48,29 @@ if(ROCMLIR_DRIVER_XDLOPS_TEST_ENABLED)
   set(MLIR_XDLOPS "-x2")
 endif()
 
+# Passed to lit.site.cfg.py.in to set the %feature flag for the mlir E2E tests.
+set(ROCMLIR_FEATURE "")
+# mfma feature
+if(ROCMLIR_DRIVER_FORCE_MFMA)
+  set(ROCMLIR_FEATURE "-mfma=on")
```

Comment:
Can we make this uniform with the other ones, where they take `${ROCMLIR_FEATURE}`?

Come to think it, can we make this be `ROCMLIR_GEN_FLAGS` and have it be a user-controllable feature so we don't have all the booleans?

Or we can keep the booleans as well. I'd say `ROCMLIR_GEN_FLAGS` as a CMake testing setup is perhaps a good call